sdk/core/core-client-rest/src/restError.ts

```diff
@@ -18,14 +18,14 @@ export function createRestError(
   response?: PathUncheckedResponse,
 ): RestError {
   const resp = typeof messageOrResponse === "string" ? response! : messageOrResponse;
-  const internalError = resp.body.error || resp.body;
+  const internalError = resp.body?.error || resp.body;
```

Comment:
nit: maybe we can use `??`

Reply:
yeah I was wondering about that too. I thought maybe there was a reason we were using || (maybe for the case where `error` is empty string we want to capture the whole body?) so thought safer to leave it as is

Reply:
actually that doesn't make sense since we assume it's an object below. I think ?? probably is more sensible then.